sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,246 @@
+# September'21 release notes
+
+Release notes for commit range 4fc5ebe..bd68232
+
+## New features
+### SYCL Compiler
+ - Disabled kernel function propagating up the call tree to callee when in
+   SYCL 2020 mode [2667e3e]
```

Comment:
I wouldn't call it a new feature. To me it looks like a bug fix. If anything, we have removed functionality from the compiler  in that patch

Reply:
Agree, but it doesn't sound like a bug fix as well. Does "improvements" section sound good for this for you?

Reply:
> Does "improvements" section sound good for this for you?

Yeah, that sounds better, i.e. we are one step closer to be compliant with SYCL 2020 spec